ansible/roles/csm.rr.mgmt_nodes_placement_validation/tasks/main.yml

```diff
@@ -40,3 +40,8 @@
   delegate_to: "{{ play_hosts | first }}"
   run_once: true
   changed_when: mgmt_nodes_placement_validation.rc == 0
+
+- name: Skip rest of play if placement validation condition is not met
+  run_once: true
+  meta: end_play
+  when: "'false' in mgmt_nodes_placement_validation.stdout"
```

Comment:
Using this check means that if 'false' appears anywhere in the output, it will match this. I worry that this could end up happening by accident. I think you would be wiser to use something less likely to encounter that problem. Like `RACK_RESILIENCY_REQUIREMENTS_NOT_MET`